ngraph/MNIST.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
What is this shell script for?

Reply:
runs 4 jobs in sequence
- /home/tensorflow/MNIST.py
- get_node_encapsulate_map.py
- import_pb_to_tensorboard.py
- ngtf_graph_viewer.py
- /usr/local/bin/tensorboard

though I suppose this could be moved into a locally defined jobs component that runs jobs sequentially. I probably should of done this all along, it would of perhaps justified the jobs component